packages/components/src/internal/components/forms/input/RadioGroupInput.tsx

```diff
@@ -23,84 +67,63 @@ interface OwnProps {
 
 type RadioGroupInputProps = OwnProps & FormsyInjectedProps<any>;
 
-interface State {
-    selectedValue: string;
-}
+const RadioGroupInputImpl: FC<RadioGroupInputProps> = memo(props => {
```

Comment:
nit: This component needs a `displayName`

Reply:
done